src/control/cmd/daos_server/command_test.go

```diff
@@ -0,0 +1,198 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/common/test"
+	"github.com/daos-stack/daos/src/control/logging"
+	"github.com/google/go-cmp/cmp"
+	"github.com/jessevdk/go-flags"
+	"github.com/pkg/errors"
```

Comment:
nit: Import ordering.

Reply:
can I please make the change in the follow-up PR: https://github.com/daos-stack/daos/pull/10022 so that we can make progress and get this one landed

Reply:
done in PR-10022